src/aioamazondevices/api.py

```diff
@@ -487,7 +487,8 @@ async def call_alexa_info_skill(
         info_skill_name: str,
```

Comment:
```suggestion
        info_skill_name: list[ALEXA_INFO_SKILLS],
```

or something like that so you don't have to check if the parameter is correct